hack/make/release-deb

```diff
@@ -16,6 +16,7 @@ set -e
 
 : ${DOCKER_RELEASE_DIR:=$DEST}
 APTDIR=$DOCKER_RELEASE_DIR/apt/repo
+GPG_NAME_EMAIL=${GPG_NAME_EMAIL:-releasedocker}
```

Comment:
This can be simplified to just:
```bash
: ${GPG_NAME_EMAIL:=releasedocker}
```
(like `DOCKER_RELEASE_DIR` above it)

Can we also call this instead `GPG_KEYID` so that it matches the wording for the `-k` option in the `dpkg-sig` manpage? (http://manpages.debian.org/dpkg-sig)

```
       -k keyid
           Specify the key ID to be used for signing; overrides any -e or -m
           option.
```